Content.Server/GameObjects/Components/Recycling/RecyclerComponent.cs

```diff
@@ -185,5 +215,31 @@ void ICollideBehavior.CollideWith(IEntity collidedWith)
         {
             Recycle(collidedWith);
         }
+
+        public SuicideKind Suicide(IEntity victim, IChatManager chat)
+        {
+            victim.PopupMessageOtherClients(Loc.GetString("{0:theName} tries to recycle {0:themself}!", victim));
+            victim.PopupMessage(Loc.GetString("You recycle yourself!"));
+
+            if (victim.TryGetComponent<IBody>(out var body))
+            {
+                foreach (var part in body.Parts.Values)
+                {
+                    if (!body.TryDropPart(part, out var dropped))
+                    {
+                        continue;
+                    }
+
+                    foreach (var drop in dropped)
+                    {
+                        drop.Owner.Delete();
+                    }
+                }
+            }
```

Comment:
This one's kinda jank ingame when I use it so I'll ping smug coz I'm not sure what it SHOULD do @DrSmugleaf 

Rest is fine.

Reply:
Ideally gib or recycle, but as it stands now delete yes.

Reply:
Although, should the parts even be deleted after being ""gibbed""? It might be better to do away entirely with deleting entities with the recycler for now.

Reply:
Removed the entity deletion part.